executor/executor.go

```diff
@@ -729,7 +763,10 @@ func (e *HashJoinExec) doJoin(idx int) {
 	e.wg.Done()
 }
 
-func (e *HashJoinExec) join(ctx *hashJoinCtx, bigRow *Row) bool {
+// joinOneBigRow creates result rows from a big table Row and send them to resultRows channel.
```

Comment:
->JoinOneBigRow creates result rows from row in a big table and sends them to the resultRows channel.